packages/migration/src/migrations/application-config/20230301124048-add-marriage-certificate-configuration.ts

```diff
@@ -17,15 +17,22 @@ export const up = async (db: Db, client: MongoClient) => {
   const session = client.startSession()
   try {
     await session.withTransaction(async () => {
-      await db.collection('certificates').insertOne({
-        event: 'marriage',
-        status: 'ACTIVE',
-        svgCode: marriageCertificateTemplateDefault,
-        svgDateCreated: Number(new Date()),
-        svgDateUpdated: Number(new Date()),
-        svgFilename: 'farajaland-marriage-certificate-v1.svg',
-        user: 'j.campbell'
-      })
+      const existingCollections = await db
```

Comment:
Could you elaborate on what this logic is doing exactly?

Reply:
This is basically adding the marriage certificate if the birth & death certificates exists